.github/workflows/build-v80.yml

```diff
@@ -0,0 +1,85 @@
+name: Build PHP V8.0
+
+on:
+  push:
+    branches: [ main ]
+
+# Ensure existing runs are cancelled automatically if still in progress when a
+# new one commit is pushed to this pull request. This is scoped to the branch
+# and workflow name so that we can run multiple workflows on a branch but
+# ensure they're exclusive with regards to concurrency.
+concurrency:
+  group: ${{ github.head_ref }}-${{ github.workflow }}
+  cancel-in-progress: true
+
+env:
+  # Use docker.io for Docker Hub if empty
+  REGISTRY: ghcr.io
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+      # This is used to complete the identity challenge
+      # with sigstore/fulcio when running outside of PRs.
+      id-token: write
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      # Install the cosign tool except on PR
+      # https://github.com/sigstore/cosign-installer
+      - name: Install cosign
+        uses: sigstore/cosign-installer@6e04d228eb30da1757ee4e1dd75a0ec73a653e06
+        with:
+          cosign-release: 'v1.13.1'
+
+      # Workaround: https://github.com/docker/build-push-action/issues/461
+      - name: Setup Docker buildx
+        uses: docker/setup-buildx-action@885d1462b80bc1c1c7f0b00334ad271f09369c55
+
+      # Login against a Docker registry except on PR
+      # https://github.com/docker/login-action
+      - name: Log into registry ${{ env.REGISTRY }}
+        uses: docker/login-action@465a07811f14bebb1938fbed4728c6a1ff8901fc
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      # Extract the current version of PHP in Dockerfile
+      - name: Extract Current Version
+        run: |
+          FILE_VER_STRING=$(sed -n '7p' < ./8.0/alpine3.16/fpm-nginx/Dockerfile)
+          VER_TRIM_START=$(echo $FILE_VER_STRING | cut -d' ' -f2)
+          VER_TRIM_END=$(echo $VER_TRIM_START | cut -d'-' -f1)
+          VER_REMOVE_COLON=$(echo $VER_TRIM_END | sed 's/://')
+          echo "CURRENT_VERSION=$VER_REMOVE_COLON" >> $GITHUB_ENV
+
+      # Does the Image exist
+      - name: Does the image exist?
+        run: |
+          # run command and capture stderr
+          docker pull ${{ env.REGISTRY }}/${{ github.event.repository.owner.login }}/${{ github.event.repository.name }}:${{ env.CURRENT_VERSION }} 2> errors.txt || true
+          echo ::set-output name=errors::`cat errors.txt`
```

Comment:
When looking at the build details I saw [this warning about `set-output` being deprecated and removed soon](https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/)